app/database/queries/breeze_buddy/analytics.py

```
@@ -0,0 +1,282 @@
+        conditions.append(f'lct."status" = ${len(values) + value_offset}')
+
+    if "outcome" in filters and filters["outcome"]:
+        values.append(filters["outcome"])
+        conditions.append(f'lct."outcome" = ${len(values) + value_offset}')
+
+    if "request_id" in filters and filters["request_id"]:
+        values.append(filters["request_id"])
+        conditions.append(f'lct."request_id" = ${len(values) + value_offset}')
+
+    # Generic payload filters (JSONB queries)
+    if "payload_filters" in filters and filters["payload_filters"]:
```

Ruff (0.14.8)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)